pkg/yurthub/proxy/proxy.go

```diff
@@ -105,7 +108,11 @@ func (p *yurtReverseProxy) buildHandlerChain(handler http.Handler) http.Handler
 	}
 	handler = util.WithMaxInFlightLimit(handler, p.maxRequestsInFlight)
 	handler = util.WithRequestClientComponent(handler)
+
+	handler = util.WithSaTokenSubsitute(handler, p.tenantToken)
```

Comment:
yurthub should not enable `WithSaTokenSubsitute` handler in a default mode, maybe we can add a startup parameter for yurthub that used to decide to enable this feature or not.